renderer/reducers/app.js

```diff
@@ -105,9 +105,9 @@ export const initDatabase = () => async dispatch => {
   try {
     await initDb()
     dispatch({ type: INIT_DATABASE_SUCCESS })
-  } catch (e) {
-    dispatch({ type: INIT_DATABASE_FAILURE, initDatabaseError: e })
-    dispatch(showError(`Unable to initialise database: ${e.message}`, { timeout: 0 }))
+  } catch (error) {
+    dispatch({ type: INIT_DATABASE_FAILURE, error })
```

Comment:
Not related to this pr, but i've noticed that in other places we usually use e.message in X_FAILURE actions

Reply:
Good catch. I found several places where we pass the full error object instead of just the message and fixed as part of this PR. I must have missed this one. Will update.

Reply:
Updated